### PR TITLE
FHB-381 Fix admin UI warnings

### PR DESCRIPTION
**Pull request by lazcool**
Fix warnings in Manage UI.
Also contains:
* set direct nuget package vulnerability checking globally (at least for the time being)
* some bonus tidy ups in Connect UI

### CODE REVIEW

File: src/ui/manage-ui/src/FamilyHubs.ServiceDirectory.Admin.Web/Areas/AccountAdmin/Pages/ManagePermissions/Edit.cshtml.cs [hunk 1]
```diff
@@ -64,11 +64,6 @@ private async Task<string> GetOrganisationName(AccountDto account)
             var organisationId = long.Parse(organisationClaim.Value);
             var organisation = await _serviceDirectoryClient.GetOrganisationById(organisationId);
 
-            if (organisation == null)
```
Comment (StuwiiDev) [inline src/ui/manage-ui/src/FamilyHubs.ServiceDirectory.Admin.Web/Areas/AccountAdmin/Pages/ManagePermissions/Edit.cshtml.cs:67]: Should this exception have been changed to the InvalidOperationException? same as you've done elsewhere when replacing Exceptions.

Comment (lazcool) [reply src/ui/manage-ui/src/FamilyHubs.ServiceDirectory.Admin.Web/Areas/AccountAdmin/Pages/ManagePermissions/Edit.cshtml.cs:67]: It seemed about the best existing exception to use.

We could create new exceptions, but these exceptions aren't explicitly caught anywhere, so using InvalidOperationException seemed appropriate to fix the warning.

What would you suggest?

Comment (StuwiiDev) [reply src/ui/manage-ui/src/FamilyHubs.ServiceDirectory.Admin.Web/Areas/AccountAdmin/Pages/ManagePermissions/Edit.cshtml.cs:67]: I am referring to the removal of the `throw new Exception("No organisation matches organisationId in claim");` block in this file. It's fine, I was just curious as other areas you just changed the type of exception

Comment (lazcool) [reply src/ui/manage-ui/src/FamilyHubs.ServiceDirectory.Admin.Web/Areas/AccountAdmin/Pages/ManagePermissions/Edit.cshtml.cs:67]: Ah gotcha! 😄 
I removed the catch blocks when I thought that it didn't add any extra context (that you couldn't get from the call stack).

**Review by StuwiiDev — APPROVED** [review]

**Review by Top-Cat — APPROVED** [review]

**Pull request merged**